src/GrumPHP/Collection/FilesCollection.php

```diff
@@ -73,6 +73,22 @@ public function path($pattern)
     }
 
     /**
+     * Filter by paths
+     *
+     * $collection->paths(array('/^spec\/','/^src\/'))
+     *
+     * @param array $patterns
+     *
+     * @return FilesCollection
+     */
+    public function paths(array $patterns)
```

Comment:
@veewee Yes you are right I refactor the path() function to call paths()    
I forgot this one array, I have changed it.    
What do you mean spec test, a new function testing it or just add the new line in `\spec\GrumPHP\Task\PhpcsSpec::it_should_have_configurable_options` ?